content/engineering-practices-why-metrics.md

```diff
@@ -0,0 +1,41 @@
+Title: Why Metrics?
+Date: 2024-04-26
+Category: Posts 
+Tags: azure-devops, tips, engineering, code-quality, automation
+Slug: engineering-practices-metrics
+Author: Martin Lacey
+Summary: Why should I be concerned about metrics as long as my code works?
+
+
+# Set the Stage
+
+No matter what stage of your career or size of business you are working with – and especially if you have aspirations of starting your own business or being partner in a startup, metrics are your key to success.  Let me explain why.
+> ![Teams collaborating driven by metric](/images/engineering-practices-why-metrics-1.png)
+
+When I first began software engineering, as a hobbyist – it was always about performance – how fast could what I built process the input and provide accurate output.  I would change the code, look for ways to optimize, and try again.  How could I tell I was moving in the right direction?  You guessed it – metrics.  It became a challenge of its own, to best my previous notions on good and fast logic – and how to use the latest capabilities of the language I was using (at the time, Rocky Mountain Basic – circa 1977).
+> ![Developing driven by metric](/images/engineering-practices-why-metrics-2.png)
```

Comment:
@MerlinDarkstar  please add an empty line before and after the > callout, otherwise html formatting goes weird.